src/code/z_prenmi.c

```diff
@@ -39,17 +39,21 @@ void PreNMI_Draw(PreNMIContext* prenmiCtx) {
     CLOSE_DISPS(gfxCtx, "../z_prenmi.c", 112);
 }
 
-void PreNMI_Main(PreNMIContext* prenmiCtx) {
+void PreNMI_Main(GameState* thisx) {
+    PreNMIContext* prenmiCtx = (PreNMIContext*)thisx;
```

Comment:
Maybe `this` should be used in this file, if you want to do that

Reply:
I can do that. I would like to do it for z_play as well, what do you think

Reply:
Really not sure about z_play.c tho, that's globalCtx so I think it would be confusing right now

Reply:
I see it as no different than the other gamestates in terms of how confusing it is, but its not that big of a deal. we can change it when we rename globalCtx